mass_mailing_list_dynamic/models/mail_mass_mailing_contact.py

```diff
@@ -9,13 +9,15 @@
 class MassMailingContact(models.Model):
     _inherit = "mail.mass_mailing.contact"
 
-    @api.constrains("partner_id", "list_id", "name", "email")
+    @api.constrains("partner_id", "list_ids", "name", "email")
     def _check_no_manual_edits_on_fully_synced_lists(self):
         if self.env.context.get("syncing"):
             return
-        if any((one.list_id.dynamic and
-                one.list_id.sync_method == "full") for one in self):
-            raise ValidationError(
-                _("Cannot edit manually contacts in a fully "
-                    "synchronized list. Change its sync method or execute "
-                    "a manual sync instead."))
+        for one in self:
+            if any((
+                list_id.dynamic and list_id.sync_method == "full"
+            ) for list_id in one.mapped('list_ids')):
+                raise ValidationError(
+                    _("""Cannot edit manually contacts in a fully synchronized list.
+                        Change its sync method or execute
+                        a manual sync instead."""))
```

Comment:
If you don't touch this string, we will not have to translate it again.

Reply:
I'm not editing things for giggles, if this changed it would be from running `black` or other linting/clean up as prescribed by the documentation.

Reply:
Probably was an error on my part, however this file was renamed to  `mailing_contact.py` and does not have this problem.